src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -579,6 +579,13 @@ export class MatAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
         {originX: 'start', originY: 'top', overlayX: 'start', overlayY: 'bottom'}
       ]);
 
+    this._positionStrategy.positionChanges.subscribe(({connectionPair}) => {
```

Comment:
Add a comment here like...
```ts
// The overlay edge connected to the trigger has squared corners, while
// the opposite end has rounded corners. We apply a css class to swap the
// border-radius based on the overlay position.
```

Alternatively, since this seems to be a common pattern, what do you think of adding a `panelClass` property to `ConnectedPosition` (to be applied when that position is used)?

Reply:
I've added the comment and I've left a note for myself to add a `panelClass` to the `ConnectedPosition`.